pyxdf/test/test_library_basic.py

```diff
@@ -24,3 +25,31 @@ def vla(data: bytes):
     assert vla(b'\x08\xfd\x12\x00\x34\x12\x34\x56\x78') == 0x78563412340012fd
     with pytest.raises(RuntimeError):
         vla(b'\x00')
+
+
+FIXTURE_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'test_files')
```

Comment:
Can you do this with pathlib?

Reply:
I was just following the most straightforward example from pytest-datafiles. These small details aren't important until we iron out the bigger details, which may require using a completely different way of downloading/loading data.